test/e2e/metrics_test.go

```diff
@@ -30,6 +31,7 @@ func testMetrics(t *testing.T) {
 	t.Run("Windows Exporter configuration validation", testWindowsExporter)
 	t.Run("Prometheus configuration validation", testPrometheus)
 	t.Run("Windows Prometheus rules validation", testWindowsPrometheusRules)
+	t.Run("Windows Node resource usage statistics validation", testNodeResourceUsage)
```

Comment:
I suggest making this `Windows Node resource usage info validation` given how you have described the test